alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunnerIT.kt

```diff
@@ -507,6 +512,96 @@ class MonitorRunnerIT : AlertingRestTestCase() {
         assertEquals("Enabled times do not match", monitor.enabledTime, retrievedMonitor.enabledTime)
     }
 
+    fun `test monitor with throttled action for same alert`() {
```

Comment:
Throttling ITs should go into another PR. This PR is specific to schema mappings.

Reply:
My bad, please ignore these changes and review other parts first.